docs/browsers-support/README.md

```diff
@@ -24,11 +24,31 @@ We use [Babel](https://babeljs.io/) with [@babel/preset-env](https://babeljs.io/
 
 Language can be transpiled but features must be polyfilled.
 
-Install [@babel/polyfill](https://babeljs.io/docs/en/babel-polyfill) and set `@babel/preset-env` option `useBuiltIns` to `usage` to automatically load the necessary polyfills based on your code and Browserslist settings (see above).
+Install [core-js@3](https://github.com/zloirock/core-js), set `@babel/preset-env` option `useBuiltIns` to `usage` to automatically load the necessary polyfills based on your code and Browserslist settings (see above) and set the `corejs` version to `3`.
+
+The [@babel/plugin-transform-runtime](https://babeljs.io/docs/en/babel-plugin-transform-runtime) plugin should also be used to enable the re-use of Babel's injected helper code and save on codesize.
+
+```js
+// babel.config.js
+
+module.exports = {
+  presets: [
+    ["@babel/preset-env", {
+      useBuiltIns: "usage",
+      corejs: 3,
+    }]
+  ],
+  plugins: [
+    ["@babel/transform-runtime", {
```

Comment:
I think the correct value is `@babel/plugin-transform-runtime`

Reply:
Yep, sorry for that.